npm WARN generates a Maven ERROR

In my npm install phase some node_modules generate some warnings about deprecated versions (like grunt). But in the Maven log, this is an ERROR instead of a WARNING. We're using Jenkins as our build tool and these errors cause wrong log analysis.

```
[INFO] --- frontend-maven-plugin:0.0.27:npm (install node, npm and dependencies) @ theme ---
[INFO] Found proxies: []
[INFO] Running 'npm install --color=false' in C:\theme
[ERROR] npm WARN deprecated lodash@0.9.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
[ERROR] npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
[ERROR] npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
[ERROR] npm WARN prefer global node-gyp@3.2.1 should be installed with -g
```

Changing the logging from ERROR to WARNING would be nice.

--- a/frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/InputStreamHandler.java
+++ b/frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/InputStreamHandler.java
@@ -53,7 +53,12 @@ final class InputStreamHandler extends Thread {
         return new LogLevelAgnosticLogger() {
             @Override
             public void log(String value) {
-                logger.error(value);
+                // fix #343
+                if (value.startsWith("npm WARN ")) {
+                    logger.warn(value);
+                } else {
+                    logger.error(value);
+                }
             }
         };
     }

